src/Features/Core/Portable/AddRequiredParentheses/AddRequiredParenthesesCodeFixProvider.cs

```diff
@@ -58,7 +58,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 // Do not add the simplifier annotation.  We do not want the simplifier undoing the 
                 // work we just did.
                 editor.ReplaceNode(node,
-                    (current, _) => syntaxFacts.Parenthesize(
```

Comment:
`ISyntaxFactsService.Parenthesize` method was redundant and has been removed in favor of `SyntaxGenerator.AddParentheses`, both of them end up invoking the same extension method at the end.